Yoast/wordpress-seo
# Deprecate the WPSEO_Cornerstone class

## Summary

This PR can be summarized in the following changelog entry:

* Deprecation of the `WPSEO_Cornerstone` class which isn't used anymore

## Relevant technical choices:

* 

## Test instructions

This PR can be tested by following these steps:

* Check if cornerstone field still can be saved on the post edit page
* Verify the cornerstone filter still works on overview pages.

## Quality assurance

* [x] I have ~added~ removed unittests to verify the code works as intended

Fixes #10626
